cypress/integration/cucumber/step_definitions/refapp-3.x/06-patient-search/patient-search.js

```diff
@@ -1,6 +1,22 @@
-import {Given} from 'cypress-cucumber-preprocessor/steps';
+import {Given, When, Before, After} from 'cypress-cucumber-preprocessor/steps';
 
-Given('the user login to the Registration Desk', () => {    
+let identifier = null;
+let patient = null;
+let patientName = `Patient${Math.random().toString(36).substring(2,7)}`;
+
+
```

Comment:
isn't one empty line enough to separate declared variable from methods?
```suggestion
```

Reply:
Yes @kdaud, I'll remove it

Reply:
have you taken a look at the empty lines [here](https://github.com/openmrs/openmrs-contrib-qaframework/blob/115a62fab0a4248a11eda5010a885317d9884f65/cypress/integration/cucumber/step_definitions/refapp-3.x/06-patient-search/patient-search.js#L6-L7)?

Reply:
I removed the extra line, as suggested